### PR TITLE
Emoji

### PR DESCRIPTION
**Pull request by caroleolivier**
This is my favourite post so far I think, enjoy :wink:

### CODE REVIEW

File: _posts/2017-11-08-adding-emoji-support.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+date: 2017-11-08
+title: Adding GitHub-flavored emoji support to my website
+---
+
+Today, I looked into an important problem: the lack of support for GitHub like emojis in my website.
```
Comment (simonproffitt) [inline _posts/2017-11-08-adding-emoji-support.md]: I think github-like needs a hyphen

Comment (simonproffitt) [reply _posts/2017-11-08-adding-emoji-support.md]: ~in~ _on_ my website

File: _posts/2017-11-08-adding-emoji-support.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+date: 2017-11-08
+title: Adding GitHub-flavored emoji support to my website
+---
+
+Today, I looked into an important problem: the lack of support for GitHub like emojis in my website. 
+
+My website, the one you are on right now, is compiled by [Jekyll](https://jekyllrb.com/) and hosted on [GitHub](https://github.com/). By default this setup doesn't support GitHub like emojis in Markdown files :scream:. By that I mean that writing `:+1:` doesn't automatically convert to :+1:.
```
Comment (simonproffitt) [inline _posts/2017-11-08-adding-emoji-support.md]: Github-like

File: _posts/2017-11-08-adding-emoji-support.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+##### Emojis Unicode
+
+Bear in mind that it is still possible to use emojis by directly adding the unicode character to the text, e.g. [🔥]. If you don't see a fire between the [] it means your browser (or whatever you are using to view this text) doesn't support [Unicode 6.0](https://emojipedia.org/unicode-6.0/). However, it is not very easy to use, unless you have an emoji keyboard or know the emoji unicode table  by heart :heart:.
+
+
+##### GitHub emojis
+
+I followed this GitHub :octocat: [tutorial](https://help.github.com/articles/emoji-on-github-pages/) but I couldn't run my site locally anymore as I didn't have the [jemoji](https://github.com/jekyll/jemoji) plugin installed.
+<br/>
+This actually got me digging and looking into this other GitHub [tutorial](https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/) about setting up your GitHub Pages locally. I comtemplated following it for a while, i.e. use [Bundler](http://bundler.io/) to manage my website dependencies, but I didn't want :x: to learn about a new software today and be distracted from my main task: GitHub emojis support.
```
Comment (simonproffitt) [inline _posts/2017-11-08-adding-emoji-support.md]: ~use~ _using_ Bundler

Comment (simonproffitt) [reply _posts/2017-11-08-adding-emoji-support.md]: ~a~ _any_ new software

File: _posts/2017-11-08-adding-emoji-support.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+I haven't looked at the source code of jemoji, but inspecting my webpage after "compilation", it looks like jemoji transforms strings like `:elephant:` to HTML image elements.
+```
+:elephant:
+```
+becomes
+```html
+<img class="emoji" title=":elephant:" alt=":elephant:"
+src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f418.png" height="20" width="20">
+```
+So adding 30 or so emojis like I did above result into 30 (very important) HTTP calls :phone: over the network.
```
Comment (simonproffitt) [inline _posts/2017-11-08-adding-emoji-support.md]: Results ~into~ _in_

File: _posts/2017-11-08-adding-emoji-support.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+
+##### Conclusion
+
+This is really cool :+1: I can now write professional looking posts :bowtie: and really convey my emotions behind each line :smiley: :sob: else how could you possibly know how I feel? 🤔 (<= not a GitHub emoji yet :cry:)
```
Comment (simonproffitt) [inline _posts/2017-11-08-adding-emoji-support.md]: Probably _the_ rather than ~my~ emotions

**Review by simonproffitt — CHANGES_REQUESTED** [review]
This is perhaps the most important work in computer science this decade. I smell a Turing award coming your way, after a few grammatical fixes :)

File: _posts/2017-11-08-adding-emoji-support.md (hunk 1 above)
Comment (caroleolivier) [reply _posts/2017-11-08-adding-emoji-support.md]: argh, I changed it 3 times haha

File: _posts/2017-11-08-adding-emoji-support.md (hunk 4 above)
Comment (caroleolivier) [reply _posts/2017-11-08-adding-emoji-support.md]: oops

**Review by simonproffitt — APPROVED** [review]

**Pull request merged**